src/Backend/Modules/ContentBlocks/Domain/ContentBlock/ContentBlock.php

```diff
@@ -1,16 +1,15 @@
 <?php
 
-namespace Backend\Modules\ContentBlocks\Entity;
+namespace Backend\Modules\ContentBlocks\Domain\ContentBlock;
 
 use Backend\Core\Engine\Model;
-use Backend\Modules\ContentBlocks\ValueObject\ContentBlockStatus;
 use Common\Locale;
 use DateTime;
 use Doctrine\ORM\Mapping as ORM;
 
 /**
  * @ORM\Table(name="content_blocks")
- * @ORM\Entity(repositoryClass="Backend\Modules\ContentBlocks\Repository\ContentBlockRepository")
+ * @ORM\Entity(repositoryClass="Backend\Modules\ContentBlocks\Domain\ContentBlock\ContentBlockRepository")
```

Comment:
you should try it first, but I think because they are in the same namespace you can drop the FQCN and just use `ContentBlockRepository`